Mathlib/Algebra/Category/HopfAlgCat/Basic.lean

```diff
@@ -24,6 +24,7 @@ variable (R : Type u) [CommRing R]
 
 /-- The category of `R`-Hopf algebras. -/
 structure HopfAlgCat where
+  private mk ::
```

Comment:
I added this to make it more similar to other cases where `mk` is made private. Since this category depends on the parameter `R` there is indeed a difference between `mk` and `of` here.